ckanext/tabledesigner/logic/action.py

```diff
@@ -25,23 +30,26 @@ def resource_update(
         context: Context,
         data_dict: dict[str, Any]) -> dict[str, Any]:
     res = original_action(context, data_dict)
-    _create_table_and_view(res)
+    _create_table_and_view(context, res)
     return res
 
 
-def _create_table_and_view(res: dict[str, Any]) -> None:
```

Comment:
this function is named in a way that indicates it is `_private` so context at the beginning is fine here.